hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java

```diff
@@ -1088,6 +1102,46 @@ public void testBlockThrottle(ContainerTestVersionInfo versionInfo)
     }
   }
 
+  /**
+   * The container checksum file is updated with blocks that have been deleted after the file is removed from disk,
+   * but before the transaction is removed from the DB. If there is a failure partway through, the checksum file
+   * should still get updated when the transaction is retried, even if the block file is not present.
+   */
```

Comment:
```suggestion
  /**
   * The container checksum tree file is updated with the blocks that have been deleted after the on disk block files are removed from disk,
   * but before the transaction is removed from the DB. If there is a failure partway through, the checksum tree file
   * should still get updated when the transaction is retried, even if the block file is not present.
   */
```

Reply:
Good clarifications. Update will be included in the next push.